datastream-testcommon/src/main/java/com/linkedin/datastream/testutil/DatastreamTestUtils.java

```diff
@@ -197,7 +197,6 @@ public static void updateDatastreams(ZkClient zkClient, String cluster, Datastre
    */
   public static void deleteDatastreamsNumTasks(ZkClient zkClient, String cluster, String... datastreams) {
     for (String datastreamName : datastreams) {
-      zkClient.ensurePath(KeyBuilder.datastreams(cluster));
```

Comment:
I'm assuming this change is unrelated to the current PR. Just cleanup?

Reply:
Yup